Add missing (noop) depend target when KBUILD is in use

--- driver/Makefile~	2011-10-04 11:30:18.166705768 +0200
+++ driver/Makefile	2011-10-04 11:34:07.422706055 +0200
@@ -303,6 +303,8 @@
 all : message
 	$(CMDLINE)
 
+depend:
+
 else # <<<<< USEKBUILD >>>>>>
 
 #****************************************************************************
